lptree.pas

```diff
@@ -2719,6 +2728,95 @@ function TLapeTree_InternalMethod_New.Compile(var Offset: Integer): TResVar;
   Param.Spill(1);
 end;
 
+constructor TLapeTree_InternalMethod_Operator.Create(AOperator:EOperator; ACompiler: TLapeCompilerBase; ADocPos: PDocPos = nil);
+begin
+  inherited Create(ACompiler, ADocPos);
+  if AOperator = op_UnaryMinus then 
+    self.FOperator := op_Minus
```

Comment:
This should depend on the number of parameters. Or maybe just do not support the overloading of unary operations.